src/main/java/hudson/plugins/parameterizedtrigger/BuildTrigger.java

```diff
@@ -96,7 +99,19 @@ public void buildDependencyGraph(AbstractProject owner, DependencyGraph graph) {
 		for (BuildTriggerConfig config : configs) {
 			List<AbstractProject> projectList = config.getProjectList(owner.getParent(), null);
 			for (AbstractProject project : projectList) {
-				ParameterizedDependency.add(owner, project, config, graph);
+                            if (config.triggerFromChildProjects() && owner instanceof ItemGroup) {
+                                ItemGroup<Item> parent = (ItemGroup) owner;
+                                for (Item item : parent.getItems()) {
+                                    if(item instanceof AbstractProject){
```

Comment:
I would vote against using AbstractProject in the new code, but it requires rework of `ParameterizedDependency`